xarray/backends/rasterio_.py

```diff
@@ -0,0 +1,116 @@
+from collections import OrderedDict
+import numpy as np
+
+try:
+    import rasterio
```

Comment:
Put this import inside the function where it's used below. That way you get a sensible error message if/when rasterio is not installed.